api/src/main/java/gov/cms/ab2d/api/controller/BulkDataAccessAPI.java

```diff
@@ -349,4 +350,22 @@ private String getUrlPath(Job job, String filePath) {
             return new ResponseEntity<>(null, null, HttpStatus.OK);
         }
     }
+
+    @ApiOperation(value = "A request for the FHIR capability statement", response = String.class,
+            produces = "application/json",
+            authorizations = {
+                    @Authorization(value = "Authorization", scopes = {
+                            @AuthorizationScope(description = "Returns the FHIR capability statement", scope = "Authorization") })
+            })
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Returns the FHIR capability statement", response =
+                    String.class)}
+    )
+    @ResponseStatus(value = HttpStatus.OK)
+    @GetMapping(value = "/metadata")
+    public ResponseEntity<String> capabilityStatement() {
+        CapabilityStatement capabilityStatement = new CapabilityStatement();
+        String json = new Gson().toJson(capabilityStatement);
+        return new ResponseEntity<>(json, null, HttpStatus.OK);
```

Comment:
Would it make sense to return `ResponseEntity<CapabilityStatement>`  here instead of converting it into a string?
If we could, we could remove the reference to Gson() as well.

Reply:
I'm not sure if it would serialize to JSON properly (or in a predictable way) if it's returning the class? I did it this way since it's easy to test and verify with the Gson conversion in the tests.